docs/sbctl.8.txt

```diff
@@ -113,12 +113,31 @@ EFI binary commands
                 Sign all the generated bundles.
 
 **remove-bundle** <NAME>::
-        Removes a bundle from the list.
+        Removes a bundle from the list. This does not delete the bundle itself.
 
 **list-bundles**::
         List all registered bundles to generate.
 
 
+Bundles
+-------
+
+Normally, only the kernel is signed with your secure boot keys. This leaves the
+cmdline and initramfs unprotected.
```

Comment:
```suggestion
Normally, only the kernel is signed with your secure boot keys. This means the
kernel command line and initramfs can be changed without possibility of verification.
```

or something like that? Mainly, I'd like to have `kernel command line` spelled out.